pkg/executor/docker/executor_test.go

```diff
@@ -337,14 +381,17 @@ func (s *ExecutorTestSuite) TestDockerNetworkingAppendsHTTPHeader() {
 		Engine(s.curlTask()).
 		BuildOrDie()
 
-	result, err := s.runJob(task)
+	executionID := uuid.New().String()
+	result, err := s.runJob(task, executionID)
 	require.NoError(s.T(), err)
-	require.Equal(s.T(), "test", result.STDOUT, result.STDOUT)
+	require.Equal(s.T(), executionID, result.STDOUT, result.STDOUT)
 }
 
+/*
```

Comment:
Why is this commented out? The test is about jobs timing out – presumably we still expect that to work? If the method by which we do that has changed, then just update this to test the new method.

Reply:
fixed in latest commit.